### ISSUE TITLE
Pre-built Path for OCO2 files

### ISSUE THREAD

**Issue by Mitchell-Odili** [sagarlimbu0/OCO2-OCO3#36]
Hi Sagar,

I'm looking at the notebook `Conversion of NETCDF to csv`. Do you mind helping me with the directory you used to make the pre-built paths. Thanks

**Comment by sagarlimbu0** [comment]
Hi Mitchell,

I believe you are looking at this notebook:
https://github.com/sagarlimbu0/OCO2-OCO3/blob/main/Conversion_NETCDF_to_CSV_2019_2020.ipynb

This is just an example of reading individual 'netCDF' from the list of paths for multiple 'netCDF' files. 

The '.txt' file reads the path of individual files stored locally (i.e. on my laptop). Initially, I had to download multiple files from EarthData.
I have provided the screenshot with detail.

Sagar
![Screenshot (7)](https://user-images.githubusercontent.com/60721866/184448849-b8052f99-f262-4e50-a172-6545e3ed3a80.png)
